### PR TITLE
Changelog for Rust 1.61 :octopus:

### PR DESCRIPTION
**Pull request by xFrednet**
roses are red,
violets are blue,
this changelog,
is no longer a todo!

---

changelog: none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,107 @@ document.
...
+  [#8440](https://github.com/rust-lang/rust-clippy/pull/8440)
+  [#8547](https://github.com/rust-lang/rust-clippy/pull/8547)
+* [`match_single_binding`], [`match_same_arms`], [`match_as_ref`], [`match_bool`]: Not longer
+  lint `match` expressions with `cfg`ed arms
+  [#8443](https://github.com/rust-lang/rust-clippy/pull/8443)
+* [`single_component_path_imports`]: No longer lint on macros
+  [#8537](https://github.com/rust-lang/rust-clippy/pull/8537)
+* [`ptr_arg`]: Allow `&mut` arguments for `Cow<_>`
+  [#8552](https://github.com/rust-lang/rust-clippy/pull/8552)
+* [`needless_borrow`]: No longer lints for method calles
```
Comment (giraffate) [inline CHANGELOG.md:87]: nits
```suggestion
* [`needless_borrow`]: No longer lints for method calls
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,107 @@ document.
...
+
+* [`new_without_default`]: No longer lints for `new()` methods with `#[doc(hidden)]`
+  [#8472](https://github.com/rust-lang/rust-clippy/pull/8472)
+* [`transmute_undefined_repr`]: No longer lints for single field structs with `#[repr(C)]`,
+  generic parameters, wide pointers, unions, tuples and allow several forms of type erasure
+  [#8425](https://github.com/rust-lang/rust-clippy/pull/8425)
+  [#8553](https://github.com/rust-lang/rust-clippy/pull/8553)
+  [#8440](https://github.com/rust-lang/rust-clippy/pull/8440)
+  [#8547](https://github.com/rust-lang/rust-clippy/pull/8547)
+* [`match_single_binding`], [`match_same_arms`], [`match_as_ref`], [`match_bool`]: Not longer
```
Comment (giraffate) [inline CHANGELOG.md:80]: nits
```suggestion
* [`match_single_binding`], [`match_same_arms`], [`match_as_ref`], [`match_bool`]: No longer
```

**Review by giraffate — APPROVED** [review]
It looks good, thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,107 @@ document.
...
+
+* [`redundant_slicing`]: Fixed suggestion for a method calls
+  [#8218](https://github.com/rust-lang/rust-clippy/pull/8218)
+* [`map_flatten`]: Long suggestions will now be split up into two help messages
+  [#8520](https://github.com/rust-lang/rust-clippy/pull/8520)
+* [`unnecessary_lazy_evaluations`]: Now shows suggestions for longer code snippets
+  [#8543](https://github.com/rust-lang/rust-clippy/pull/8543)
+* [`unnecessary_sort_by`]: Now suggests `Reverse` including the path
+  [#8462](https://github.com/rust-lang/rust-clippy/pull/8462)
+* [`search_is_some`]: More suggestions are now `MachineApplicable`
```
Comment (flip1995) [inline CHANGELOG.md:104]: Is there no PR for this? Otherwise LGTM

Comment (xFrednet) [reply CHANGELOG.md:104]: It has and now I've also added it :upside_down_face:

**Review by flip1995 — APPROVED** [review]
Thanks, please squash the 2 commits and then r=me :+1:

**Pull request merged**